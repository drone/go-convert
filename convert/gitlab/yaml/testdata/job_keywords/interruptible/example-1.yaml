# https://docs.gitlab.com/ee/ci/yaml/#interruptible

stages:
  - stage1
  - stage2
  - stage3

step-1:
  stage: stage1
  script:
    - echo "Can be canceled."
  interruptible: true

step-2:
  stage: stage2
  script:
    - echo "Can not be canceled."

step-3:
  stage: stage3
  script:
    - echo "Because step-2 can not be canceled, this step can never be canceled, even though it's set as interruptible."
  interruptible: true