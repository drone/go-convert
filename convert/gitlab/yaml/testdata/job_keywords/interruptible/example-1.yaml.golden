stages:
    - stage1
    - stage2
    - stage3
step-1:
    interruptible: true
    script:
        - echo "Can be canceled."
    stage: stage1
step-2:
    script:
        - echo "Can not be canceled."
    stage: stage2
step-3:
    interruptible: true
    script:
        - echo "Because step-2 can not be canceled, this step can never be canceled, even though it's set as interruptible."
    stage: stage3
